akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/aeron/AeronStreamConcistencySpec.scala

```diff
@@ -27,6 +27,9 @@ import akka.testkit._
 import akka.util.ByteString
 
 object AeronStreamConsistencySpec extends MultiNodeConfig {
+  // important to not use aeron-udp via system property override because that will cause port conflict
+  System.setProperty("akka.remote.artery.transport", "tcp")
```

Comment:
ugly, yeah, but the multi-node testkit will otherwise always honor system properties over other config due to ConfigFactory.load (and that is the right thing)